pkiintergration-fe/src/services/operation.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { IOperation } from '@c8y/client';
+import { OperationService } from '@c8y/ngx-components/api';
+
+@Injectable({ providedIn: 'root' })
+export class yourService {
+  constructor(private operationService: OperationService) {}
+
+  async requestPubKey(deviceId: string): Promise<IOperation> {
+    // http://resources.cumulocity.com/documentation/websdk/client/interfaces/ioperation.html
```

Comment:
nitpick: only keep documenting comments

remove comments if their intent is unclear, or make their use understood